plugins/weather.go

```diff
@@ -24,6 +24,19 @@ func (Weather) Triggers() []string {
 
 func (Weather) Execute(m *irc.Message) (string, error) {
 	parsed := strings.SplitN(m.Trailing(), " ", 2)
+
+	// TODO: AAAAAAAAAAAAAAAAAAAAAAAAAAA
```

Comment:
This plugin interferes with .wiki sporadically. I'm assuming the init() function ordering at build/startup is effectively pseudorandom, sort of like C++ Static Init Ordering problem?